package.json

```diff
@@ -101,7 +101,6 @@
     "@types/express-ws": "^3.0.0",
     "@types/fetch-mock": "^7.3.1",
     "@types/fs-extra": "^9.0.6",
-    "@types/got": "^9.4.4",
```

Comment:
Do we still want types?

Reply:
In this case, no. `got` includes its own types starting in version 10+; thus, no more `@types/got` dependency 😄